client_test.go

```diff
@@ -69,15 +78,63 @@ func ExampleNewClient() {
 	}
 
 	// use a devnull connector for example purposes
-	conn := &devnull.Connector{}
+	conn, err := dosaRenamed.GetConnector("devnull", nil)
+	if err != nil {
+		panic(err)
```

Comment:
nit: suggest returning `err` instead of `panic` since user may copy/paste this code; I think @junchaow suggested this when `ExampleNewClient` was first implemented.